web/src/components/profile/ProfileAddressFields.tsx

```diff
@@ -0,0 +1,113 @@
+import { ModifiedContent } from "@/components/ModifiedContent";
+import { StateDropdown } from "@/components/StateDropdown";
+import { WithErrorBar } from "@/components/WithErrorBar";
+import { AddressMunicipalityDropdown } from "@/components/data-fields/address/AddressMunicipalityDropdown";
+import { AddressTextField } from "@/components/data-fields/address/AddressTextField";
+import { AddressContext } from "@/contexts/addressContext";
+import { useAddressErrors } from "@/lib/data-hooks/useAddressErrors";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { useMountEffectWhenDefined } from "@/lib/utils/helpers";
+import { isOwningBusiness } from "@businessnjgovnavigator/shared/domain-logic/businessPersonaHelpers";
+import { ReactElement, useContext } from "react";
+
+export const ProfileAddressFields = (): ReactElement => {
```

Comment:
[sand] I HATE that this implies we're going to be adding 3 new address components, but I think in the name of clarity we should change this name.

```suggestion
export const ProfileNJAddress = (): ReactElement => {
```